use-cases/rag-pipeline/alloy-db-setup/src/get_emb.py

```diff
@@ -12,11 +12,14 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import asyncio
+import aiohttp
 import logging
 import logging.config
 import os
 import requests
 import json
+import backoff
```

Comment:
Organize your imports, there is built in functionality to do this in Code with the Python extension.